src/OpenTK/Platform/Linux/LinuxNativeWindow.cs

```diff
@@ -374,6 +374,11 @@ public override Point PointToScreen(Point point)
             return new Point(point.X + client.X, point.Y + client.Y);
         }
 
+        public override void ConfineCursor (bool confine)
+        {
+            throw new NotImplementedException ();
```

Comment:
IIRC calling many of the non-relevant methods on IOS/ Android produces a NotImplementedException, so there is form for doing things this way.

It depends though, as this could conceivably be called at any time during runtime (as opposed to at init where you'd definitely want the exception IMHO)

Any maintainers with an opinion?